sys/include/net/netopt.h

```diff
@@ -147,6 +147,12 @@ typedef enum {
                                  *   the current state */
     NETOPT_AUTOACK,             /**< en/disable link layer auto ACKs or read
                                  *   the current state */
+    NETOPT_ACK_PENDING,         /**< en/disable the frame pending bit of ACKs.
```

Comment:
Please also add this option [here](https://github.com/RIOT-OS/RIOT/blob/d15c7559f342985a64c969f8e6d93eb650a8503e/sys/net/crosslayer/netopt/netopt.c#L26).

Reply:
Done. Thanks!